### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by jaydlomb**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by jaydlomb** [comment]
Locked into the pull request comment

**Review by jaydlomb — COMMENTED** [review]
Code is looking alright,
Bit blocky, add some spaces and indents to better format code

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (jaydchamp) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Looks good! maybe delete the emojis

Comment (jaydlomb) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

**Pull request merged**